tests/initial_condition_SAVANI.prm

```diff
@@ -0,0 +1,82 @@
+set Dimension                              = 3
+set Use years in output instead of seconds = false
+set Start time                             = 0
+set End time                               = 0
+
+
+subsection Material model
+  set Model name = simple
+
+  subsection Simple model
+    set Thermal expansion coefficient = 4e-5
+    set Viscosity                     = 1e22
+    set Reference temperature         = 1600
+  end
+end
+
+
+subsection Geometry model
+  set Model name = spherical shell
+
+  subsection Spherical shell
+    set Inner radius  = 3481000
+    set Outer radius  = 6336000
+  end
+end
+
+
+subsection Model settings
+  set Zero velocity boundary indicators       = 0
+  set Tangential velocity boundary indicators = 1
+  set Prescribed velocity boundary indicators =
+
+  set Fixed temperature boundary indicators   = 0,1
+
+  set Include shear heating                   = false
+end
+
+
+subsection Boundary temperature model
+  set Model name = spherical constant
+  subsection Spherical constant
+    set Inner temperature = 1600
+    set Outer temperature = 1600
+  end
+end
+
+
+subsection Initial conditions
+  set Model name = SAVANI perturbation
+  subsection SAVANI perturbation
+    set Initial condition file name           = savani.dlnvs.60.m.ab
+    set vs to density scaling                 = 0.3
```

Comment:
This is unrelated to the current patch, and may be addressed separately: Could we rename this parameter so that "Vs" is capitalized? We always do that with parameters, and in this case I think there isn't even much of an argument that it might be a symbol because people will probably have an easier time reading Vs over vs when thinking of the shear wave velocity.